src/sidekick/config.py

```diff
@@ -59,6 +59,7 @@
         "tool_ignore": [
             "read_file",
         ],
+        "guide_file": "",  # When set, overrides the default GUIDE_FILE
```

Comment:
Should we hardcode this to be SIDEKICK.md so its obvs?

Reply:
good call